js/src/contract/item.ts

```diff
@@ -1292,23 +327,10 @@ export async function getItemProgram(
   env: string,
   customRpcUrl: string
 ): Promise<ItemProgram> {
-  if (customRpcUrl) log.debug("USING CUSTOM URL", customRpcUrl);
-
-  const solConnection = new web3.Connection(customRpcUrl || getCluster(env));
-
-  if (anchorWallet instanceof web3.Keypair)
-    anchorWallet = new NodeWallet(anchorWallet);
-
-  const provider = new AnchorProvider(solConnection, anchorWallet, {
-    preflightCommitment: "recent",
-  });
-
-  const idl = await Program.fetchIdl(ITEM_ID, provider);
-
-  const program = new Program(idl, ITEM_ID, provider);
-
-  return new ItemProgram({
-    id: ITEM_ID,
-    program,
-  });
+  return Program.Program.getProgramWithWallet(
```

Comment:
Just an FYI, when we update the CLI to use sol-command you can get rid of this `getItemProgram` function. Until then, you can instead call the `getProgramWithWallet` from the version that is inherited from Program, `ItemProgram.getProgramWithWallet`. Slightly less verbose API than `Program.Program`